internal/transform/config_setting.go

```diff
@@ -143,8 +138,11 @@ func TransformConfigSetting(ledgerChange ingest.Change, header xdr.LedgerHeaderH
 		LastModifiedLedger:              uint32(ledgerEntry.LastModifiedLedgerSeq),
 		LedgerEntryChange:               uint32(changeType),
 		Deleted:                         outputDeleted,
-		ClosedAt:                        closedAt,
-		LedgerSequence:                  uint32(ledgerSequence),
+		ClosedAt:                        changeDetails.ClosedAt,
+		LedgerSequence:                  changeDetails.LedgerSequence,
+		TransactionID:                   changeDetails.TransactionID,
+		OperationID:                     changeDetails.OperationID,
+		OperationType:                   changeDetails.OperationType,
```

Comment:
i think we can actually drop transaction id, operation id and type from this table. config settings are only an internal change, configured by validators. my assumption is that these three fields will be `nil`